modules/core/05-port/keeper/keeper.go

```diff
@@ -87,7 +87,19 @@ func (k *Keeper) LookupModuleByPort(ctx sdk.Context, portID string) (string, *ca
 // Route returns a IBCModule for a given module, and a boolean indicating
 // whether or not the route is present.
 func (k *Keeper) Route(module string) (types.IBCModule, bool) {
-	return k.Router.GetRoute(module)
+	routes, ok := k.Router.GetRoute(module)
```

Comment:
This change is the equivalent of what was done by Damian here: https://github.com/cosmos/ibc-go/pull/6314/commits/16a88b4d6fd14dd2b7e0cc9ccb0103b41b809605#diff-91fb52aa2c91092fdd10a9a17c92247d443a378b977c5f808f6f0cdafedce9ac though I'm not 100% sure it applies identically here.

Reply:
The change makes sense :+1: We fixed for routing on `SendPacket` which needed to use the portID rather than the module name embedded into a capability. Now that capabilities are removed, the same needs to be done and dynamic port creation requires this prefix trick